tracer/test/Datadog.Trace.Security.IntegrationTests/IAST/Grpc/GrpcDotNetTests.cs

```diff
@@ -0,0 +1,82 @@
+// <copyright file="GrpcDotNetTests.cs" company="Datadog">
+// Unless explicitly stated otherwise all files in this repository are licensed under the Apache 2 License.
+// This product includes software developed at Datadog (https://www.datadoghq.com/). Copyright 2017 Datadog, Inc.
+// </copyright>
+
+#if NETCOREAPP3_0_OR_GREATER
+
+using System;
+using System.Linq;
+using System.Runtime.InteropServices;
+using System.Text.RegularExpressions;
+using System.Threading.Tasks;
+using Datadog.Trace.Configuration;
+using Datadog.Trace.TestHelpers;
+using VerifyXunit;
+using Xunit;
+using Xunit.Abstractions;
+
+namespace Datadog.Trace.Security.IntegrationTests.IAST.GrpcDotNet;
+
+[UsesVerify]
+public class GrpcDotNetTests : TestHelper
+{
+    public GrpcDotNetTests(ITestOutputHelper output)
+        : base("GrpcDotNet", output)
+    {
+        SetServiceVersion("1.0.0");
+        SetEnvironmentVariable(ConfigurationKeys.DebugEnabled, "1");
```

Comment:
We shouldn't test with debug enabled (it has concealed issues in the past)

Reply:
I didn't know that! Noted thanks :)
Applied in daf703e